app/components/publishers/vacancies_component.rb

```diff
@@ -30,13 +35,13 @@ def filters_applied_text
     I18n.t("jobs.dashboard_filters.heading", count: @filters[:managed_school_ids]&.count)
   end
 
-  def vacancy_type_tab_link(vacancy_type)
+  def vacancy_type_tab_link(vacancy_type, is_selected)
```

Comment:
Ruby convention tends to be not to have `is_` as part of names for booleans (so just `selected` would fit better I think)